### DISCUSSION TITLE
[Feature] Add black shadow option on fonts like on RCT Deluxe/RCT1

### DISCUSSION THREAD

**Discussion by md500** [OpenRCT2/OpenRCT2#17432] [Feature requests]
Hi guys.

Is possible to add black shadow feature like on Open RCT 1? I'm using RCT1 Theme but OpenRCT lacks black shadow option:
![Themes Comparision](https://user-images.githubusercontent.com/3111925/174504456-45d60627-708a-4b68-ad53-929621aa0922.jpg)

RCT1 Look (with black shadow):
![RCT1-LOOK](https://user-images.githubusercontent.com/3111925/174504463-3e298292-0b33-4548-8bae-d9b7fb4dbe93.jpg)

RCT2 Look (no black shadow):
![RCT2-LOOK](https://user-images.githubusercontent.com/3111925/174504470-0e4a6266-c42e-4af1-ac98-1d9f5f1aab63.jpg)

OpenRCT with RCT1 Look
![OPENRCT-w-RCT1-LOOK](https://user-images.githubusercontent.com/3111925/174504673-ab86d3d2-04a7-43d2-bb54-d21566595676.jpg)

The modify error is to assume prebuilt theme. I know about Duplicate options, but i dont see feature to add black shadow to fonts while i creating custom theme.

Is possible to replicate complete RCT1 look?

Thanks you very much!

**Comment by 733737** [comment]
it's called a text drop shadow, and it all depends if the bitmap fonts can be ported from rct deluxe without copyright issue.

**Comment by Gymnasiast** [comment]
@733737 Not true. The RCT1 and RCT2 fonts are identical, save for some added characters. The font in RCT2 (and all the characters that I have added) still has this information, there is just no code path to trigger it.

**Reply by 733737** [reply]
I thought bitmap fonts wouldn't allow dropshadows. IMO I would prefer dropshadows over no dropshadows

**Reply by Gymnasiast** [reply]
RCT fonts use indexed images.
0 = transparent
1 = main glyph
2 = shadow
3= rest of the outline

You can see this for yourself in most included glyphs in the resources/g2 folder.

**Comment by 733737** [comment]
will drop shadows be enabled for rct2?

**Reply by Gymnasiast** [reply]
For OpenRCT2, you mean? Perhaps. It shouldn’t be too difficult to modify the code to use it, at any rate.

**Reply by 733737** [reply]
I still think that drop shadows make fonts look nicer and more legible
